src/core_plugins/kibana/public/management/sections/settings/advanced_import.js

```diff
@@ -0,0 +1,40 @@
+import { uiModules } from 'ui/modules';
+import advancedImportTemplate from 'plugins/kibana/management/sections/settings/advanced_import.html';
+import { AdvancedImportState } from 'plugins/kibana/management/sections/settings/advanced_import_state.js';
+
+uiModules.get('apps/management')
+  .directive('advancedImport', function (config, Private) {
+    const advancedImportState = Private(AdvancedImportState);
```

Comment:
You're not using any angular DI, is there a reason you're wrapping this in `Private`?